.github/workflows/testing-env-image.yml

```diff
@@ -29,6 +29,9 @@ jobs:
         image: tonistiigi/binfmt:latest
         platforms: arm64,arm
 
+    - name: Set DOCKER_CONTENT_TRUST
+      run: export DOCKER_CONTENT_TRUST=1
```

Comment:
Home come dct needs to be enabled? Are we taking advantage of this feature?

Reply:
It was suggested by dockle, it ensures that we can only pull signed repositories and images in our docker build. We'll need to see if it doesn't complain about pushing to ghcr.

Reply:
In order to sign the container, you may need to also bring in a notary with a private key. Otherwise I do not think this will do anything at the moment. Would it be possible to do this in a follow up PR?

If you're unsure if it will push to ghcr, it would be good to do this in a fork to ensure uploading the image and pulling the image is successful.

Reply:
Perhaps we can remove this change from the PR and address DCT in a future PR? But first focus on getting rid of root 👍🏼

Reply:
yes this makes sense to me. Let's punt on DCT then. Thank you!

Reply:
Removed DCT